src/background/vvppManager.ts

```diff
@@ -66,9 +81,28 @@ export class VvppManager {
   ): Promise<{ outputDir: string; manifest: EngineManifest }> {
     const nonce = new Date().getTime().toString();
     const outputDir = path.join(this.vvppEngineDir, ".tmp", nonce);
+
+    const streams: fs.ReadStream[] = [];
```

Comment:
このブロックが何をしようとしてるか若干だけ見えづらいので、一言コメント書いとくとコード見る人が心構えできるかもです。
```suggestion
    // 番号順のファイル（[prefix].001.vvpp, [prefix].002.vvpp, ...）があれば連結
    const streams: fs.ReadStream[] = [];
```